repository/repository-server/src/main/java/org/eclipse/vorto/repository/core/IModelRepository.java

```diff
@@ -127,4 +127,32 @@
 	 * @return
 	 */
 	Optional<FileContent> getFileContent(ModelId modelId, String fileName);
-}
+	
+	/**
+	 * Attaches the given file to the model
+	 * 
+	 * @param modelid The modelId where to attach the file
+	 * @param fileName the filename
+	 * @param content the content of the file
+	 * @param userContext the user context
+	 * @return
+	 */
+	boolean attachFile(ModelId modelid, String fileName, byte[] content, IUserContext userContext);
+	
+	/**
+	 * Gets the list of attachments for the model
+	 * 
+	 * @param modelId
+	 * @return
+	 */
+	List<String> getAttachments(ModelId modelId);
```

Comment:
What does this method return ? Attachment file names , keys, ? Please rename the method better.